src/app/[locale]/about/page.tsx

```diff
@@ -0,0 +1,101 @@
+'use client';
+
+import Image from 'next/image';
+import { useTranslations } from 'next-intl';
+
+import favicon from '@/app/icon.svg';
+import { classNames } from '@/utils/classNames';
+
+const sectionClasses = 'space-y-6 mb-10';
+const headingClasses = 'text-2xl lg:text-3xl font-semibold text-theme-primary';
+const linkClasses = 'text-green-600 hover:underline font-semibold';
+
+export default function AboutPage() {
+  const t = useTranslations('AboutPage');
+  const courseName = useTranslations('CourseNames')('rsSchoolReact');
```

Comment:
You can use `getTranslations` instead of `useTranslations` - it will help you keep your AboutPage a server component:

```suggestion
import Image from 'next/image';
import { getTranslations } from 'next-intl/server';

import favicon from '@/app/icon.svg';
import { classNames } from '@/utils/classNames';

const sectionClasses = 'space-y-6 mb-10';
const headingClasses = 'text-2xl lg:text-3xl font-semibold text-theme-primary';
const linkClasses = 'text-green-600 hover:underline font-semibold';

export default async function AboutPage() {
  const t = await getTranslations('AboutPage');
  const courseName = (await getTranslations('CourseNames'))('rsSchoolReact');
```